words/words.md

```diff
@@ -5,7 +5,7 @@
         	Wie geht es weiter?
        	</div>
 		<div style="font-weight: 500;top: 140px;left: 10px;font-size: 29px;">
-			Zukunftsszenarien von COVID-19, erklärt mit spielbaren Simulationen
+			Zukunftsszenarien von COVID-19, erklärt mit interaktiven Simulationen
```

Comment:
Das Original hat "playable simulations", obwohl auch im Englischen "interactive simulations" viel gängiger ist. Würde daher das "spielbar" behalten.
Nicky sieht die Sachen als Spiele und schreibt in deren Kontext auch von Spieleentwicklung, das Wort wurde also recht sicher mit Bedacht gewählt.

Reply:
Von mir aus in Ordnung, wollen wir das aber dann komplett so durchziehen (z.B. L29, wo es jetzt noch "interaktiv" heißt) oder das im Laufe des Textes auch mal abwechseln?

Reply:
Ehrlich gesagt finde ich interaktive auch besser vom Sinn, aber natürlich ist das nicht die 1:1-Übersetzung von _playable_,... mmmh.

Reply:
Ich würde es konsequent mit "spielbar" durchziehen. 

@herrhundt Warum macht denn "interaktive Simulation" mehr Sinn als "spielbare Simulation"?

Reply:
Es wirkt nur geläufiger, das ist alles. Von mir aus kann auch "spielbar" durchgezogen werden.

Reply:
Alles klar, ich passe spielbar überall an.